deployment/cloud-run/api_config_production.py

```
@@ -0,0 +1,116 @@
+        "max_concurrent_requests": 20,  # Increased from 5
+
+        # Abuse detection (More lenient)
+        "rapid_fire_threshold": 30,     # Increased from 10
+        "sustained_rate_threshold": 600, # Increased from 200
+        "rapid_fire_window": 1.0,
+        "sustained_rate_window": 60.0,
+
+        # Disable some strict checks for production
+        "enable_user_agent_analysis": False,
+        "enable_request_pattern_analysis": False,
+    }
```

Ruff (0.12.2)

Mutable class attributes should be annotated with `typing.ClassVar`

(RUF012)